types/plotly.js/index.d.ts

```diff
@@ -282,6 +282,12 @@ export interface Layout {
 	bargroupgap: number;
 	selectdirection: 'h' | 'v' | 'd' | 'any';
 	hiddenlabels: string[];
+	polar: {
```

Comment:
based on the docs there are many more properties than `bgcolor` and `radialaxis.color`. Could you make this feature complete? Something like:

```ts
polar: Partial<LayoutPolar>;
```

https://plot.ly/javascript/reference/#layout-polar

Also, as is, both of those properties would be required when polar is defined, as the `?` is missing on each.

Reply:
Sure thing. Will get on it.